### ISSUE TITLE
Clipboard / Drive redirection

### ISSUE THREAD

**Issue by muff10** [danielguerra69/alpine-xfce4-xrdp#3]
Hi!

I can only get drive redirection to work if i run it in privileged mode. How do I get it to work in a non privileged mode?
Essentially I need copy/paste of files to work. Copy/paste of text works just fine.

**Comment by danielguerra69** [comment]
I don't think it is possible. File copy/paste depends on fuser and fuser needs to run privileged
